### PR TITLE
Update to 1.15.2, fix server ping if server-icon.png is not present

### PR DESCRIPTION
**Pull request by SamB440**
- [x] Update packets
- [x] Update fields
  - [x] Join Game
  - ~~[ ] Chunk Data (Biomes are now 3D)~~

### CODE REVIEW

File: src/main/java/me/minidigger/minicraft/protocol/MiniPacketRegistry.java [hunk 1]
```diff
@@ -2,40 +2,13 @@
 
 import com.google.common.collect.HashBasedTable;
 import com.google.common.collect.Table;
-
+import me.minidigger.minicraft.protocol.client.*;
+import me.minidigger.minicraft.protocol.server.*;
```
Comment (MiniDigger) [inline src/main/java/me/minidigger/minicraft/protocol/MiniPacketRegistry.java]: plz no

File: src/main/java/me/minidigger/minicraft/server/MiniServerPacketHandler.java [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+import me.minidigger.minicraft.api.Server;
+import me.minidigger.minicraft.model.*;
+import me.minidigger.minicraft.model.chunk.ChunkData;
+import me.minidigger.minicraft.model.chunk.ChunkPosition;
+import me.minidigger.minicraft.netty.MiniConnection;
+import me.minidigger.minicraft.protocol.MiniPacketHandler;
+import me.minidigger.minicraft.protocol.PacketDirection;
+import me.minidigger.minicraft.protocol.PacketState;
+import me.minidigger.minicraft.protocol.client.*;
+import me.minidigger.minicraft.protocol.server.*;
```
Comment (MiniDigger) [inline src/main/java/me/minidigger/minicraft/server/MiniServerPacketHandler.java]: same, no star imports plz

**Review by MiniDigger — CHANGES_REQUESTED** [review]
minor things, if fixed we can just merge. doesnt need to be fully implemented

File: src/main/java/me/minidigger/minicraft/server/MiniServerPacketHandler.java (hunk 2 above)
Comment (SamB440) [reply src/main/java/me/minidigger/minicraft/server/MiniServerPacketHandler.java]: I blame IntelliJ!

**Comment by MiniDigger** [comment]
thanks!

**Pull request merged**